linked_list/linked_list.py

```diff
@@ -55,4 +84,10 @@ def search(self, value):
         :param value: The item searched for
         :returns: True if the value was found, or False if the value was not found 
         """
-        pass
+        current_node = self.head
+        while current_node:
+            if current_node.value == value:
+                return True
+            else:
+                current_node = current_node.next
```

Comment:
You don't really need the else block because of the return statement above

```suggestion
            if current_node.value == value:
                return True
            current_node = current_node.next
```